backend/test/remove_institution_handler_test.py

```diff
@@ -27,7 +27,7 @@ def test_post(self):
         # Verify the members
         admin = mocks.create_user()
         common_user = mocks.create_user()
-        institution = mocks.create_institution()
+        institution = mocks.create_institution(has_address=True)
```

Comment:
This approach will be hard to maintain as the mock grows. It must be generic or the property is set outside the mock creation.

```
institution = mocks.create_institution()
institution.address = Address()
```